### PR TITLE
Add firstAired to query parameters

### PR DESCRIPTION
**Pull request by dskvr**
Missing a number of other parameters for query as well

See [forum topic](https://forums.thetvdb.com/viewtopic.php?p=89441#p89441)

### CODE REVIEW

**Comment by vankasteelj** [comment]
Hi, thanks but you also need to add the firstAired field into the "optional" array, so it won't become a mandatory parameter

File: methods.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
         "pagination": true
     },
     "/series/episodes/query": {
-        "url": "/series/:id/episodes/query?absoluteNumber=&airedSeason=&airedEpisode=&dvdSeason=&dvdEpisode=&imdbId=",
+        "url": "/series/:id/episodes/query?absoluteNumber=&airedSeason=&airedEpisode=&dvdSeason=&dvdEpisode=&imdbId=&firstAired=",
         "pagination": true,
         "optional": ["absoluteNumber", "airedSeason", "airedEpisode", "dvdSeason", "dvdEpisode", "imdbId"]
```
Comment (vankasteelj) [inline methods.json]: firstAired should be in the array of optional parameters as it is not a mandatory one.

**Review by vankasteelj — CHANGES_REQUESTED** [review]

File: methods.json (hunk 1 above)
Comment (dskvr) [reply methods.json]: @vankasteelj Resolved

**Review by vankasteelj — APPROVED** [review]

**Comment by vankasteelj** [comment]
Merged, thanks

**Pull request merged**